File: 1431-kids-with-the-greatest-number-of-candies/1431-kids-with-the-greatest-number-of-candies.java
class Solution {
    public List<Boolean> kidsWithCandies(int[] candies, int extraCandies) {
        List<Boolean> list=new ArrayList<>();
        int mx=0;
        for (int i: candies){
            mx=Math.max(mx,i);
        }
        for (int j: candies){
            if (mx>(j+extraCandies))
                list.add(false);
            else
                list.add(true);
        }return list;
    }
}

File: 1431-kids-with-the-greatest-number-of-candies/1431-kids-with-the-greatest-number-of-candies.py
class Solution:
    def kidsWithCandies(self, c: List[int], extra: int) -> List[bool]:
     
        a=max(c)
        for i in range(len(c)):
            
            if c[i]+extra>=a:
                c[i]=True
            else:
                c[i]=False
        return c
                
            
        